lib/build.js

```diff
@@ -88,35 +82,25 @@ const renderHTML = ({
       const { ServerStyleSheet } = require('styled-components')
       const sheet = new ServerStyleSheet()
       html = render(
-        sheet.collectStyles(
-          React.createElement(App.default, { routes, path })
```

Comment:
Need to make sure the `createElement` function isn't needed inside `sheet.collectStyles`